src/utils/index.js

```diff
@@ -0,0 +1,21 @@
+import childrenValueInputValidation from './childrenValueInputValidation';
+import createChainedFunction from './createChainedFunction';
+import createContextWrapper from './createContextWrapper';
+import CustomPropTypes from './CustomPropTypes';
+import deprecationWarning from './deprecationWarning';
+import domUtils from './domUtils';
+import EventListener from './EventListener';
+import assign from './Object.assign';
+import ValidComponentChildren from './ValidComponentChildren';
+
+export default {
+  childrenValueInputValidation,
+  createChainedFunction,
+  createContextWrapper,
+  CustomPropTypes,
+  deprecationWarning,
+  domUtils,
+  EventListener,
```

Comment:
If EventListener is being removed from this namespace I think it should be removed from the libraries internals entirely and specified as a dependency.

Object.assign should be included via the ES6 -> ES5 build step.